ext/pdo/tests/gh8626.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+GH-8626: PDOStatement->execute() failed, then execute successfully, errorInfo() information is incorrect
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo')) die('skip');
+$dir = getenv('REDIR_TEST_DIR');
+if (false == $dir) die('skip no driver');
+if (!strncasecmp(getenv('PDOTEST_DSN'), 'mysql', strlen('mysql'))) die('skip not relevant for mysql driver');
```

Comment:
Could use the better readable
```suggestion
if (str_starts_with(getenv('PDOTEST_DSN'), 'mysql')) die('skip not relevant for mysql driver');
```
However, I wonder why the test is not relevant for PDO_MySQL.